specifyweb/frontend/js_src/lib/components/Interactions/InteractionDialog.tsx

```diff
@@ -161,7 +163,10 @@ export function InteractionDialog({
           loanPreparation.set('quantityResolved', 0);
         });
         itemCollection.add(item);
```

Comment:
There looks to be a bug in here (was present before). When the code follows this branch the dialog is not closed

Not sure if that's related to https://github.com/specify/specify7/pull/3389/files

Reply:
If you are interested, here is how that code looked like before I rewrote it to typescript and react - https://github.com/specify/specify7/blob/b5bde53cb6a3178212752b2e4e31949888778cd8/specifyweb/frontend/js_src/lib/interactiondialog.js#L288-L298